src/zns.ts

```diff
@@ -57,10 +60,14 @@ export default class Zns extends NamingService {
     this.network = source.network as string;
     this.url = source.url as string;
     if (!this.network) {
-      throw new Error('Unspecified network in Resolution ZNS configuration');
+      throw new ConfigurationError(ConfigurationErrorCode.UnspecifiedNetwork, {
```

Comment:
Do we consider this a breaking change?

Reply:
Shouldn't be, before it was a simple error now it is a Configuration Error which extends Error. These errors were designed to be raised to tell a developer that something is wrong before they could launch. Meaning they were not supposed to be caught